packages/core/bootstrap/src/lib/cache-warmer/reducer.ts

```diff
@@ -50,9 +55,12 @@ export interface SubscriptionState {
 export const subscriptionsReducer = createReducer<SubscriptionState>({}, (builder) => {
   builder.addCase(actions.warmupSubscribed, (state, { payload }) => {
     const key = payload.key || getSubscriptionKey(payload)
-
+    const { data, ...rest } = payload.data 
```

Comment:
Should be able to remove this now.

Reply:
Wait don't we need this as we are sending payload like

```
{
  data: {
     data: {}
  }
}
```

or are we updating the action's payload?